runtime/vm/jvminit.c

```diff
@@ -6308,7 +6308,10 @@ parseXlogForCompatibility(J9JavaVM *vm)
 							if ('\0' == *upToComma) {
 								goto xlogerr;
 							}
-							gclog = 0 != j9_cmdla_stricmp("off", upToComma);
+							/* only levels trace, debug, info, and not off, warning, error, enable the gc verbose log */
+							gclog = (0 == j9_cmdla_stricmp("trace", upToComma))
+								|| (0 == j9_cmdla_stricmp("debug", upToComma))
+								|| (0 == j9_cmdla_stricmp("info", upToComma));
 							if (gclog) {
```

Comment:
I think this should continue to recognize "off"; and the change should be the addition of recognizing additional levels including which enable logging:
```
	/* only levels trace, debug, info, and not off, warning, error, enable the gc verbose log */
	if (0 == j9_cmdla_stricmp("off", upToComma)
	|| (0 == j9_cmdla_stricmp("warning", upToComma))
	|| (0 == j9_cmdla_stricmp("error", upToComma))
	) {
		gclog = FALSE;
	} else if ((0 == j9_cmdla_stricmp("trace", upToComma))
			|| (0 == j9_cmdla_stricmp("debug", upToComma))
			|| (0 == j9_cmdla_stricmp("info", upToComma))
	) {
		gclog = TRUE;
	} else {
		/* unrecognized log level */
		unrecognizedOption = TRUE;
	}
```

Reply:
I've made a change along these lines. OpenJ9 doesn't really understand log levels, so I think it's appropriate to set unrecognizedOption for every level except "off". Perhaps we should even set it for `-Xlog:gc` but that seems too much.